src/main/client/player/player-sprite.ts

```diff
@@ -21,33 +26,33 @@ interface PlayerOptions {
    };
 }
 
-const MAX_HORIZONTAL_SPEED = 3;
+const MAX_HORIZONTAL_SPEED = 1;
 const MAX_VERTICAL_SPEED = 10;
-export const PLAYER_HEIGHT = 40;
+export const PLAYER_HEIGHT = 20;
 export const PLAYER_WIDTH = 10;
 
 export class PlayerSprite extends Phaser.GameObjects.Container {
    private prevPosition = new Vector2();
    private positionChangedSubject = new Subject<Vector2>();
    readonly positionChanged$ = this.positionChangedSubject.asObservable();
-   private gun: GunSprite;
-   private character: Phaser.GameObjects.Sprite;
+   private readonly gun: GunSprite;
+   private readonly character: Phaser.GameObjects.Sprite;
    private jumping = false;
    private verticalSpeed = 0;
-   private debugger: HitBoxDebugger;
+   private readonly debugger: HitBoxDebugger;
 
    constructor(private readonly options: PlayerOptions) {
       super(options.scene, 0, 0);
+      this.setScale(1 / ClientConfig.MAP_OUTPUT_SCALE, 1 / ClientConfig.MAP_OUTPUT_SCALE);
       const config = {
-         key: 'player1-walk',
-         frames: options.scene.anims.generateFrameNumbers('player1', { frames: [0, 1, 2, 6, 7, 8] }),
+         key: 'player-walk',
```

Comment:
player1-walk is better, i am planning to add another player sprite, so they all wont look the same

![p1_stand](https://user-images.githubusercontent.com/2917613/99906816-1ceccc80-2cd1-11eb-930d-2d87a7270025.png)
![p2_front](https://user-images.githubusercontent.com/2917613/99906818-1f4f2680-2cd1-11eb-85a8-a669062816f2.png)
![p3_front](https://user-images.githubusercontent.com/2917613/99906823-21b18080-2cd1-11eb-901a-5c0483e4b8d2.png)

Reply:
Fixed.